CHANGELOG.md

```diff
@@ -39,10 +39,9 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Improvements
 
-* (tools) [#14793](https://github.com/cosmos/cosmos-sdk/pull/14793) Dockerfile optimization.
```

Comment:
Makes sense, we'll need to create a PR targeting main too with the same removals, otherwise, when the changelog is added to main this will never get deleted from main (as it won't appear in any release).

Reply:
ill go through and audit main as well.